src/pages/mine.tsx

```diff
@@ -27,16 +28,23 @@ const Mine: FC = () => {
     response,
     error,
     isLoading,
-  } = useFetch(TASKS_URL);
-  useEffect(() => { setTasks(response); }, [isLoading, response]);
+    callAPI
+  } = useFetch(TASKS_URL, {}, false);
+  const { isLoggedIn, isLoading: isAuthenticating } = useAuthenticated();
```

Comment:
could we move useAuthenticated to top level (_app) that way we don't have to call it again and again for each page.

Reply:
As it is used in just two pages, it wouldn't be a good idea to do so